### PR TITLE
Added warning about the usage of the custom schema

### PR DESCRIPTION
**Pull request by pictos**
### Description of Change ###

Added more info about the usage of the custom schema xmlns.

 ### Linked Issues ###
 

 - Related to #427

 ### PR Checklist ###
 
 - [ ] Has a linked Issue, and the Issue has been `approved`(bug) or `Championed` (feature/proposal)
 - [ ] Has tests (if omitted, state reason in description)
 - [ ] Has samples (if omitted, state reason in description)
 - [ ] Rebased on top of `main` at time of PR
 - [ ] Changes adhere to [coding standard](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#contributing-code---best-practices)
 - [ ] Documentation created or updated: https://github.com/MicrosoftDocs/CommunityToolkit/pull/XYZ 

 ### Additional information ###

### CODE REVIEW

**Comment by bijington** [comment]
Thanks @pictos I really like the idea! I would like us to provide some information around the differences between using a library and app project and give an example of how to overcome it, perhaps this should go in the NuGet read me file too.

I'm hoping to watch that video on the subject so I have a better understanding before I make some full suggestions though.

File: src/CommunityToolkit.Maui/ReadMe.txt [hunk 1]
```diff
@@ -24,10 +24,14 @@ In order to make use of the toolkit within XAML you can use this namespace:
 
 xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
 
+!!!WARNING!!!
+In order to use the custom schema mentioned above you need to call the `UseMauiCommunityToolkit` method or call any API on C# code. If you don't do that that url will not work and you need to use the `xmlns= namespace;assembly` pattern. This is a limitation of the custom schema API, more info here:
```
Comment (VladislavAntonyuk) [inline src/CommunityToolkit.Maui/ReadMe.txt:28]: could we also add it to this page as well: https://docs.microsoft.com/en-us/dotnet/communitytoolkit/maui/get-started

**Review by VladislavAntonyuk — APPROVED** [review]

**Review by brminnick — CHANGES_REQUESTED** [review]
I prefer that we fix the problem by adding a Source Generator  instead of adding a warning to the docs that (let's be honest) most developers won't read

**Comment by pictos** [comment]
@brminnick how do you feel about merging this and creating an issue to use an SG to fix the issue?
If you go on our discord channel, you'll see some points that we need to test and make sure that SG will work as expected.

**Comment by bijington** [comment]
> @brminnick how do you feel about merging this and creating an issue to use an SG to fix the issue?
> If you go on our discord channel, you'll see some points that we need to test and make sure that SG will work as expected.

If we do merge I would like to reword it a bit to make it a little bit more helpful in guiding developers on how to workaround the issue.

**Comment by pictos** [comment]
@bijington do you have something in mind already? If so feel free to push the changes to this PR

**Comment by bijington** [comment]
> @bijington do you have something in mind already? If so feel free to push the changes to this PR

Yeah I can happily provide something on this

**Comment by bijington** [comment]
Sorry I should add my content to this! I'll try and do that this weekend

**Comment by brminnick** [comment]
`.UseMauiCommunityToolkit()` Analyzer added in #486

**Comment by pictos** [comment]
@brminnick should we close this? Or still is good to have this in some "docs"?

**Comment by brminnick** [comment]
I'd like to add implement two more things to the code base before we close this:
1. A `CommunityToolkit.Maui.Init()` method that a user can call from a class library 
  - This provides a work-around for devs using the .NET MAUI Community Toolkit in a class library to
  - The method wouldn't actually do anything (eg it wouldn't wire-up handlers), but it ensures that they have written C# code from `CommunityToolkit.Maui` and the custom schema will not fail
2. An analyzer that verifies a developer as called `CommunityToolkit.Maui.Init()` in their class library

  ```cs
  public static void Init()
  {
    
  }
  ```

Having this PR open is a good reminder that we still need to do it!

**Comment by brminnick** [comment]
Superseded by https://github.com/CommunityToolkit/Maui/pull/486

**Pull request closed without merging**